src/gna-lib/kernels/KernelArguments.h

```diff
@@ -109,7 +109,14 @@ struct KernelConfig : public BaseConfig
 
 struct ExecutionConfig
 {
-    ExecutionConfig() = default;
+    const ExecutionConfig& operator =(const ExecutionConfig& right)
+    {
+	*this->SaturationCount = *right.SaturationCount;
+	return *this;
```

Comment:
Why only `SaturationCount` field is assigned ?
Where is this `operator=` used ?

consider to ` = delete` if possible